src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -292,6 +294,12 @@ const TrackingWrappedArtworkApp: React.FC<Props> = props => {
 
   const websocketEnabled = !!sale?.extendedBiddingIntervalMinutes
 
+  const initialAlertCriteria = {
+    attributionClass: compact([attributionClass?.internalID]),
+    artistIDs: compact(artists).map(artist => artist.internalID),
+    additionalGeneIDs: [mediumType?.filterGene?.slug as string],
+  }
```

Comment:
Initial criteria is a bit more involved here including Medium Type and Rarity values.